### ISSUE TITLE
Documentation: How to change Figaro path

### ISSUE THREAD

**Issue by joallard** [laserlemon/figaro#183]
I initially commented on e8b302f, but I figure opening an issue will be better for tracking.

How do you actually change the path of `config/application.yml`? To be honest, I don't like that default, as it always collides with `config/application.rb` in autocompletion.

The tests allude to `Figaro::Application.new(path = "")`, but it's not us who actually call that function.

The documentation is silent as to how to do that

**Comment by mrjamesriley** [comment]
Seems there's no configuration allowed for this, for the Rails usage. The `figaro install` allows you to pass in the path, e.g `figaro install -p config/passport.yml` - but this only creates the file, and appends it to the git ignore. But then the gem will still look for the file at `config/application.yml`.

I had a quick test and this works:

```Figaro.application = Figaro::Rails::Application.new(path: Rails.root.join('config', 'passport.yml'))
Figaro.load```

Although it's more of a workaround, as `Figaro.load` would be getting called for the second time here.

**Comment by joallard** [comment]
Yup, it's the workaround I ended up using for the time being. The crux is that because Figaro runs on before_configuration, you can't even config it with an initializer. I wonder where we could make space to put some settings.

**Comment by joallard** [comment]
Also to note that the param in the workaround will be ignored by `figaro heroku:set`

**Comment by joallard** [comment]
What if, dirty fix, we checked if there was a `config/initializers/figaro.rb` and loaded that before the cli

**Comment by joallard** [comment]
@laserlemon Would you accept a PR that would run `config/initializers/figaro.rb` on load if present? Dirty fix, but I can't think of another strategy

**Comment by mikeymurph77** [comment]
+1
